src/general/OptionsMenu.tscn

```diff
@@ -825,6 +825,13 @@ margin_right = 511.0
 margin_bottom = 27.0
 rect_min_size = Vector2( 300, 0 )
 
+[node name="ScreenShot folder" type="Button" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Misc/VBoxContainer"]
+margin_top = 349.0
+margin_right = 511.0
+margin_bottom = 379.0
+action_mode = 0
+text = "Open Screenshot Folder"
```

Comment:
This needs to be changed to use the new translations system.

Reply:
What am I suppose to do there?

Reply:
https://github.com/Revolutionary-Games/Thrive/blob/master/doc/working_with_translations.md#adding-new-content-into-the-game-with-translation-in-mind